ext/standard/tests/strings/implode_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test implode() function: error conditions
+--FILE--
+<?php
+/* only glue */
+try {
+    var_dump( implode("glue") );
```

Comment:
I know that this code is copied but as it changes place, could you use PSR-12 like coding style (not strictly required but that's kind of what is used for new test code). It basically means removing those spaces after `(` and before `)`...

Reply:
I didn't know if it would be ok to change those se it was just copied. Good to know it's ok. haha

I'll do it tonight and push. Thx!